src/code/z_play.c

```diff
@@ -451,14 +458,16 @@ void Play_Init(GameState* thisx) {
     AnimationContext_Update(this, &this->animationCtx);
     gSaveContext.respawnFlag = 0;
 
-    if (OOT_DEBUG && R_USE_DEBUG_CUTSCENE) {
+#if OOT_DEBUG
+    if (R_USE_DEBUG_CUTSCENE) {
         gDebugCutsceneScript = sDebugCutsceneScriptBuf;
         PRINTF("\nkawauso_data=[%x]", gDebugCutsceneScript);
 
         // This hardcoded ROM address extends past the end of the ROM file.
```

Comment:
can `sDebugCutsceneScriptBuf` be brought down as in function static to remove an ifdef? Did not check closely if the data ordering the file works out for it

Reply:
er wait it gets used somewhere else also, nevermind

Reply:
It's used only in this block I think, I tried it and it worked

Reply:
oh, i somehow misread that when going to check myself lmao. cool.